tests/test-unsigned-summaries.sh

```diff
@@ -128,16 +128,8 @@ assert_file_has_content metadata "'ostree.ref-binding': <\['ostree-metadata'\]>"
 echo "ok 6 update repo metadata"
 
 # Try to update the app, which should pull the updated repository metadata as a
```

Comment:
This comment says you're updating the app, but the line updating it is being deleted?

Reply:
Fixed.